packages/esm-patient-registration-app/src/patient-registration/field/id/identifier-selection-overlay.tsx

```diff
@@ -21,42 +21,43 @@ const PatientIdentifierOverlay: React.FC<PatientIdentifierOverlayProps> = ({
   identifiers,
   remove,
 }) => {
-  const { patientIdentifiers } = useContext(ResourcesContext);
-  const [identifierTypes, setIdentifierTypes] = useState<PatientIdentifierType[]>([]);
+  const { identifierTypes } = useContext(ResourcesContext);
+  const [localIdentifierTypes, setLocalIdentifierTypes] = useState<Array<PatientIdentifierType>>([]);
```

Comment:
What makes a local identifier type local? What's the difference between that and `identifierTypes`?

Reply:
HI @brandones! As you can see in the screenshots above, the side panel overlay contains the whole list of identifier types, but whatever changes are done in the panel remains in the panel only and is not global until the `save` button is clicked, hence I have copied the global-identifier types into the local one (local is in accordance to the side panel overlay) so it does no effect in the global ones and hence named localIdentifierTypes.
I hope it clears your doubt.

Reply:
Okay, so "global" means saved (to the server) and "local" means unsaved? Please use "saved" and "unsaved" for those concepts, not "global" and "local."

Reply:
Sure, did it as per your suggestion. I'll be more specific to the names from the next time.
Thank You!